### ISSUE TITLE
DMCA language in the Code of Conduct

### ISSUE THREAD

**Issue by krusynth** [publicinteresttown/pit-jekyll#1]
[This thread from rahaeli](https://nitter.net/rahaeli/status/1593819064161665024) has got me thinking this morning that we should explicitly prohibit posting content that you don't own the copyright to and are using outside of Fair Use. That is, memes are fine but like don't dump your mp3 collection on here. 

(We all still have mp3 collections right?)

**Comment by hartsick** [comment]
we'll need a sneakernet for mp3s :jk-foia:

I know @dariusk has had some conversations recently about legal obligations for Mastodon admins and he mentioned he'll be co-publishing some guidance around that. I think it may make sense to just wait until that's available to shape language / other practices, but it does sound like from what he shared of those conversations that we should go ahead and register a DMCA agent for the instance: https://www.brownwinick.com/insights/what-is-a-designated-dmca-agent-and-why-you-need-one

**Comment by dariusk** [comment]
Yes. It costs $6 for a registration that lasts 3 years. One note is that you must put a physical mailing address and not a PO box and it will be publicly viewable. For mine, I put the address of the small office I rent since I do receive mail there. You can apply for an exception to allow for a PO box but it takes a while to get reviewed. Also I have had some fun conversation with people familiar with the Copyright Office and how this new world of individual servers might overwhelm them and cause them to change how all of this works...

**Comment by tBaxter** [comment]
This feels like a good opportunity for some additional media guidelines, if we don't have them already (and if we do and I missed it, forgive me). I'm thinking things like "consider downsizing your photos" and "consider hosting your video elsewhere"


And I have just shy of 14,000 mp3s in my library.
